lib/srv/sess.go

```diff
@@ -39,11 +42,15 @@ import (
 	"github.com/gravitational/teleport/lib/sshutils"
 	"github.com/gravitational/teleport/lib/utils"
 
+	broadcast "github.com/dustin/go-broadcast"
```

Comment:
Can you just use a condvar and avoid pulling another dependency?

My general approach is if a dependency looks to be well maintained, supported by a large community, and significantly accelerates our development, we should use it.

On the other hand, if it's only needed in one or two places and looks like it was built/supported by one random person, we can probably get by without it.

More information/justification in #5685.

Reply:
Seems like a fair tradeoff in-line with the linked. It's removed